BranchSDK/src/BranchIO/Request.cpp

```diff
@@ -14,10 +14,10 @@ using namespace Poco;
 
 namespace BranchIO {
 
-int const Request::MaxAttemptCount = 5;
+int const Request::DefaultRetryCount = 5;
```

Comment:
The design at this point is such that it isn't just the number of retries that is important, it is the total number of seconds that it will retry.   Each retry backs off a little longer before it tries again, and if it were only 3 total (with this logic) it would fail within a single minute.